src/components/application_manager/src/commands/mobile/on_button_event_notification.cc

```diff
@@ -57,6 +57,9 @@ void OnButtonEventNotification::Run() {
       static_cast<uint32_t>(
           (*message_)[strings::msg_params][hmi_response::button_name].asInt());
 
+  ApplicationSharedPtr app = ApplicationManagerImpl::instance()->application(
+      (*message_)[strings::msg_params][strings::app_id].asUInt());
```

Comment:
@OHerasym is app-id mandatory parameter according API? if not, please check if it is exists.